### DISCUSSION TITLE
Using Dex for Openstask keystone

### DISCUSSION THREAD

**Discussion by pnatraj** [dexidp/dex#2182] [Q&A] [answered]
I looked at the connector for keystone can i use it for openstack keystone integration is there a documentation or reference that we could follow to integration of openstack keystone with dex

From the page https://github.com/dexidp/dex/ the openstack keynote link   https://dexidp.io/docs/connectors/keystone/  the page is not accessible

Page Not Found
Looks like you've followed a broken link or entered a URL that doesn't exist on this site.
Back to our site
If this is your site, and you weren't expecting a 404 for this path, please visit Netlify's "page not found" support guide for troubleshooting tips.

**Comment by nabokihms** [comment, marked as answer]
We have not merged [PR with documentation](https://github.com/dexidp/website/pull/90) yet. You can find the actual doc file here https://github.com/dexidp/website/blob/0b4b9ad331148c85f17278f875c36eb0dbafb385/content/docs/connectors/keystone.md.

**Comment by pnatraj** [comment]
Thank you very much
